dts/arm/st/f4/stm32f427vX.dtsi

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (c) 2020 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <st/f4/stm32f427.dtsi>
+
+/delete-node/ &spi5;
+
+/delete-node/ &spi6;
```

Comment:
I'm not sure why this would be needed

Reply:
f427's almost all variants have spi5 and spi6 but not the vX. I followed suit from the f429 implementation.

Reply:
Ok.